vllm/config.py

```diff
@@ -1362,6 +1363,17 @@ def __post_init__(self):
                              f"must be one of {valid_guided_backends}")
 
 
+@dataclass
+class ObservabilityConfig:
+    """Configuration for observability."""
+    otlp_traces_endpoint: Optional[str] = None
+
+    def __post_init__(self):
+        if not is_otel_installed() and self.otlp_traces_endpoint is not None:
+            raise ValueError("OpenTelemetry packages must be installed before "
+                             "configuring 'otlp_traces_endpoint'")
```

Comment:
Done. Additionally, please note that there is a similar check in the configuration for validating CLI arguments: 
https://github.com/vllm-project/vllm/blob/9fa949e268805dbb411992b79f25c9a6b7fdcb59/vllm/config.py#L1371-L1374